src/os/ui/config/betatoggleui.js

```diff
@@ -0,0 +1,101 @@
+goog.declareModuleId('os.file.BetaToggleUI');
+
+import Settings from '../../config/settings.js';
+import {ROOT} from '../../os.js';
+import Module from '../module.js';
+
+
+
+/**
+ * The column mapping settings UI directive
+ * @return {angular.Directive}
+ */
+export const directive = () => ({
+  restrict: 'E',
+  replace: true,
+  templateUrl: ROOT + 'views/file/betatoggle.html',
+  controller: Controller,
+  controllerAs: 'ctrl'
+});
+
+/**
+ * The element tag for the directive.
+ * @type {string}
+ */
+export const directiveTag = 'betatoggle';
+
+
+Module.directive(directiveTag, [directive]);
+
+
+/**
+ * Controller for beta features settings.
+ * @unrestricted
+ */
+export class Controller {
+  /**
+   * Constructor.
+   * @param {!angular.Scope} $scope
+   * @ngInject
+   */
+  constructor($scope) {
+    /**
+     * @type {?angular.Scope}
+     * @protected
+     */
+    this.scope = $scope;
+    this.Settings = Settings.getInstance();
```

Comment:
Anything that you add to `this` or to `this.scope` should have a jsdoc attached to say what the value holds (string, array, object, class type, etc) and a brief explanation of what it is. So for example this would be:
```
    /**
     * Settings instance for this class
     * @type {Settings}
     */
    this.settings  = Settings.getInstance();
```

the same needs to be done for `this['betaFeatureNames']`